.github/workflows/wheels.yml

```diff
@@ -0,0 +1,65 @@
+name: Wheels
```

Comment:
I don't think we need this pipeline either; what is it providing over the standard CI pipeline? Our medium term plan is to publish to pypi, but we've not gotten that setup just yet. Having the wheels isn't really helpful IMHO